src/IIIFPresentation/API/Features/Storage/Requests/UpdateCollection.cs

```diff
@@ -78,6 +83,7 @@ public async Task<ModifyEntityResult<FlatCollection>> Handle(UpdateCollection re
         }
 
         return ModifyEntityResult<FlatCollection>.Success(
-            collectionFromDatabase.ToFlatCollection(request.UrlRoots, settings.PageSize, items));
+            collectionFromDatabase.ToFlatCollection(request.UrlRoots, 1, settings.PageSize, total,
```

Comment:
What is `1`?

Reply:
`1` is the current page - you can't currently set the page on `Update`, so git assumes the first page for `view` - it's definitely not clear though, so will use something other than a magic number